src/main/java/core/basesyntax/service/ConvertData.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.ArrayList;
+
+public interface ConvertData {
+    public ArrayList<FruitTransaction> convert(String data);
```

Comment:
Use interfaces over exact class
no need to write `public` inside interface. 
```suggestion
    List<FruitTransaction> convert(String data);
```